beacon_node/src/cli.rs

```diff
@@ -187,6 +187,29 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                 .takes_value(true),
         )
 
+        /*
+         * Standard staking flags
+         */
+
+        .arg(
+            Arg::with_name("staking")
+                .long("staking")
+                .help("Standard option for a staking beacon node. Equivalent to \
+                `lighthouse bn --http --eth1 `. This will enable the http server on localhost:5052 \
+                and try connecting to an eth1 node on localhost:8545")
+                .takes_value(false)
+                .conflicts_with("staking-with-external-eth1-endpoint")
+        )
+        .arg(
+            Arg::with_name("staking-with-ext-eth1-endpoint")
```

Comment:
I wondering if `--staking-with-ext-eth1-endpoint ...` is really easier than `--staking --eth1-endpoint ...`? Perhaps I'm missing something though?

Reply:
My rationale for the `--staking-with-ext-eth1-endpoint` flag was that users might equate `--staking` with a sigp hosted eth1 node somehow (many seemed to think that's what `--eth1` did on discord). 
Although now that I think of it, `--staking` without the correct endpoint will fail loudly. Will remove `--staking-with..` and improve the error messages.